sky/serve/serve_utils.py

```diff
@@ -313,6 +313,32 @@ def update_service_encoded(service_name: str, version: int, mode: str) -> str:
     return common_utils.encode_payload(service_msg)
 
 
+def terminate_replica(service_name: str, replica_id: int, purge: bool) -> str:
```

Comment:
Lets add `ux_utils.print_exception_no_traceback()` for all errors in the function. It makes the console output very messy (see line 2-5).

```bash
$ ssd https --replica-id 2
Terminating replica ID 2 in 'https'. Proceed? [Y/n]: 
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/home/ubuntu/skypilot-runtime/lib/python3.10/site-packages/sky/serve/serve_utils.py", line 319, in terminate_replica
    raise ValueError(f'Service {service_name!r} does not exist.')
ValueError: Service 'https' does not exist.

sky.exceptions.CommandError: Command $([ -s ~/.sky/python_path ] && cat ~/.sky/python_path 2> /dev/null || which python3) -u -c 'from sky... failed with return code 1.
Failed to terminate the replica

The above exception was the direct cause of the following exception:

RuntimeError: Failed to terminate the replica
```

Reply:
It would be great if you can help adding this to other errors raised in this file, in a separate PR ;)